### ISSUE TITLE
use snyk scan seata server maven dependency package

### ISSUE THREAD

**Issue by caozhiyuan** [seata/seata#4706]
## Why you need it?
use snyk scan seata server maven dependency package

## How it could be?
fix a lots of Vulnerabilities

## Other related information
my seata server version is 1.5.0
```
    <dependency>
      <groupId>io.seata</groupId>
      <artifactId>seata-server</artifactId>
      <version>1.5.0</version>
    </dependency>
```
Vulnerabilities:
![image](https://user-images.githubusercontent.com/3415285/174034350-9029c14a-1231-4196-a180-fb412227b02f.png)

**Comment by slievrly** [comment]
@caozhiyuan fixed in #4650

**Comment by slievrly** [comment]
@caozhiyuan besides #4650, what other security vulnerabilities need to be reported?
